### ISSUE TITLE
Match Specific Traits From Main Folders

### ISSUE THREAD

**Issue by SamFordGH** [HashLips/hashlips_art_engine#854]
Is there a way to target specific traits inside of the parent folder? For example, if I only wanted a green body with blue eyes and a sad mouth, can I specifically target these 3 traits to be paired together in a separate growEditionSizeTo? 

Im assuming some code that will go next to the parenting folder... 

growEditionSizeTo: 1,
    layersOrder: [ 
      { name: "Body (only use green body) " } ,
      { name: "Mouth (only use sad mouth)" } ,
      { name: "Eyes (only use blue eyes)"  },

**Comment by Fr4nc3sc0O** [comment]
Have you solved your problem? I am also interested in a solution to this.

**Comment by bolshoytoster** [comment]
@Fr4nc3sc0O I think [thePeanutGalleryandCo's repo](https://github.com/thepeanutgalleryandco/create-and-mint-nft-collection) or [nftchef's repo](https://github.com/nftchef/art-engine) might have this.

**Comment by thepeanutgalleryandco** [comment]
@SamFordGH @Fr4nc3sc0O yes, you can check out my repo for filtration options on your layerconfigurations.
